code/game/objects/items.dm

```diff
@@ -829,11 +832,54 @@ GLOBAL_VAR_INIT(embedpocalypse, FALSE) // if true, all items will be able to emb
 		var/timedelay = usr.client.prefs.tip_delay/100
 		var/user = usr
 		tip_timer = addtimer(CALLBACK(src, .proc/openTip, location, control, params, user), timedelay, TIMER_STOPPABLE)//timer takes delay in deciseconds, but the pref is in milliseconds. dividing by 100 converts it.
+		var/mob/living/L = usr
+		if(istype(L) && L.incapacitated())
+			apply_outline(COLOR_RED_GRAY)
+		else
+			apply_outline()
+
+/obj/item/MouseDrop(atom/over, src_location, over_location, src_control, over_control, params)
+	. = ..()
+	remove_outline()
 
 /obj/item/MouseExited()
 	deltimer(tip_timer)//delete any in-progress timer if the mouse is moved off the item before it finishes
 	closeToolTip(usr)
+	remove_outline()
 
+/obj/item/proc/apply_outline(colour = null)
```

Comment:
`colour` is a confusing name, as it constantly conflicts with `color`.

Make this `outline_color`, or whatever it is.

Reply:
british exterminated